lib/active_model/serializer/attributes.rb

```diff
@@ -17,6 +17,8 @@ module Attributes
         def attributes(requested_attrs = nil, reload = false)
           @attributes = nil if reload
           @attributes ||= self.class._attributes_data.each_with_object({}) do |(key, attr), hash|
+            key = key.to_s.gsub(/\?\z/, '').to_sym
```

Comment:
Would it be possible to move that at https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model/serializer/attributes.rb#L57? That way it's done at the serializer definition and not everytime we call the `#attributes` method.